server/src/db/models/users.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema({
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  posts: {
+    type: Array,
+    default: [],
+    required: true,

```

Comment:
I don't think the 'required' is needed, as it gets a default if no posts are provided. Same for all following entries.

Reply:
Yep removed & tested and it still works. So updated PR has these lines removed